hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1397,13 +1397,20 @@ public void refresh(OmKeyInfo key) throws IOException {
     refreshPipeline(Arrays.asList(key));
   }
 
+  private boolean isKeyDeleted(String key, Table keyTable) {
```

Comment:
Thanks for the meaningful method. Similar checks used in [OmMetadataManagerImpl.java#L936](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java#L936) . How about moving `#isKeyDeleted()` to a common function and reuse it. Maybe into OzoneManagerUtils or OmMetadataManager or a better.

OmMetadataManagerImpl.java#L936

```
          CacheValue<OmKeyInfo> cacheValue = keyTable.getCacheValue(new CacheKey<>(kv.getKey()));
          if(cacheValue == null || cacheValue.getCacheValue() != null) {
             .....
          }
```

Reply:
Will do it in the next set of changes.